backend/clap/src/main/java/zpi/ppea/clap/dtos/SubmissionDto.java

```diff
@@ -0,0 +1,23 @@
+package zpi.ppea.clap.dtos;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class SubmissionDto {
+    private Integer submissionId;
+    private Integer year;
+    private String name;
+    private Integer durationDays;
+    private List<AssessorDto> assessors;
+    private List<RatingDto> ratings;
+
+    private Integer totalRating;
```

Comment:
We don't need a total rating per submission. Total rating should be calculated on the `Rating` level. I know we'll have more than one rating per submission, but that's fine - we'll display these separately.

Reply:
same as above, work in progress